### DISCUSSION TITLE
Irony Discord

### DISCUSSION THREAD

**Discussion by bcssov** [bcssov/IronyModManager#230] [General]
Irony doesn't have an official Discord server by itself. It's too much work to manage and stuff for me to be honest. 

Having said that this doesn't mean that Irony does not have a Discord presence. Officially I hang out on the Stellaris Modding Den where under hosted tools you can find [#irony-mod-manager](https://discord.gg/ezzCnwP7fj) channel.

Or if you'd prefer now a dedicated Discord: https://discord.gg/t9JmY8KFrV

**Comment by OldEnt** [comment]
https://discord.gg/WpeXm2h

**Reply by bcssov** [reply]
An invite is already posted in the original message.
